src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    protected Color color = new ColorSupplier().getRandomColor();
```

Comment:
you should initialize color passing its generated value to the constructors of each of the figures